db/migrate/20210910164556_convert_to_jsonb.rb

```diff
@@ -0,0 +1,45 @@
+class ConvertToJsonb < ActiveRecord::Migration[6.1]
+  def up
+    convert_to_jsonb(:accounting_accounts, :quickbooks_data)
+    convert_to_jsonb(:accounting_customers, :quickbooks_data)
+    convert_to_jsonb(:accounting_qb_departments, :quickbooks_data)
+    convert_to_jsonb(:accounting_qb_vendors, :quickbooks_data)
+    convert_to_jsonb(:accounting_sync_issues, :custom_data)
+    convert_to_jsonb(:accounting_transactions, :quickbooks_data)
+    convert_to_jsonb(:data_exports, :data)
+    convert_to_jsonb(:divisions, :custom_data)
+    convert_to_jsonb(:divisions, :locales)
+    convert_to_jsonb(:organizations, :custom_data)
+    convert_to_jsonb(:projects, :custom_data)
+    convert_to_jsonb(:response_sets, :custom_data)
+    convert_to_jsonb(:tasks, :activity_message_data)
+    convert_to_jsonb(:tasks, :custom_error_data)
```

Comment:
Had to convert response sets in order to make the migration performant. jsonb is better than json so did it for all json fields at once.

Reply:
ah, great - thank you.